shopinvader_validation_token/README.rst

```diff
@@ -0,0 +1,69 @@
+============================
+Shopinvader validation token
+============================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-shopinvader%2Fshopinvader-lightgray.png?logo=github
+    :target: https://github.com/shopinvader/shopinvader/tree/10.0/shopinvader_validation_token
+    :alt: shopinvader/shopinvader
+
+|badge1| |badge2| |badge3| 
+
+This Shopinvader addon implements a security layer on the customer mode.
+To ensure a customer doesn't use the email of somebody else,
+it generates a token/code sent using notification and
+the customer has to fill it on the front-side.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+* By interface:
+
+- Browse into your shopinvader backends;
+- Add a notification named "Security token" (for customer service);
```

Comment:
not required here but would be nice to see a backport of https://github.com/shopinvader/odoo-shopinvader/tree/13.0/shopinvader_notification_default and a new default template for this token there ;)